### PR TITLE
doc: Fix typos in COPYRIGHT

### PR DESCRIPTION
**Pull request by GChuf**
Fixed a couple of typos

### CODE REVIEW

File: contrib/debian/copyright [hunk 1]
```diff
@@ -72,7 +72,6 @@ Files: src/qt/res/icons/tx_mined.png
        src/qt/res/src/hd_enabled.svg
 Copyright: Jonas Schnelli
 License: Expat
-Comment:
```
Comment (MarcoFalke) [inline contrib/debian/copyright]: This looks like a syntax change on the file format and not a typo?

**Comment by MarcoFalke** [comment]
ACK. Correct name is indeed "Stephen Hutchings"

File: contrib/debian/copyright (hunk 1 above)
Comment (gapeman) [reply contrib/debian/copyright]: There is no typo here, just a deleted line because there was no actual comment there.

**Comment by hebasto** [comment]
> Correct name is indeed "Stephen Hutchings"

Has he account on the GutHub?

**Comment by laanwj** [comment]
>ACK. Correct name is indeed "Stephen Hutchings"

I was looking for conclusive source on this but couldn't find it

**Comment by gapeman** [comment]
https://github.com/stephenhutchings
https://stephenhutchings.com/
https://www.s-ings.com/me/

File: contrib/debian/copyright (hunk 1 above)
Comment (MarcoFalke) [reply contrib/debian/copyright]: Phrased differently: Can debian still parse the file, when this field is missing? Why was it added in the first place?

**Comment by emilengler** [comment]
Concept ACK

File: contrib/debian/copyright (hunk 1 above)
Comment (gapeman) [reply contrib/debian/copyright]: Oh - looking at the file you can see that comments aren't added every time. So I just assumed it's safe to delete this line.

Comment (MarcoFalke) [reply contrib/debian/copyright]: ok, fine

**Review by promag — COMMENTED** [review]
ACK 36f7697cf6c129cabff2a698a52115829cd0d1fb, correct name is already here:

https://github.com/bitcoin/bitcoin/blob/f418c3379cbad8caaf3e988caf727fe62b218f8c/contrib/debian/copyright#L51

All occurrences are now correct.

**Review by fanquake — APPROVED** [review]
ACK 36f7697cf6c129cabff2a698a52115829cd0d1fb

**Pull request merged**